src/dune/lib.ml

```diff
@@ -801,8 +836,12 @@ module Vlib : sig
 
     val add : t -> lib -> t Or_exn.t
 
-    val fold :
-      t -> init:'acc -> f:(lib -> 'acc -> 'acc Or_exn.t) -> 'acc Or_exn.t
```

Comment:
`fold` was used to implement the two functions below, but that made the code convoluted.